modules.yml

```diff
@@ -247,6 +259,10 @@ categories:
         slug: 'micronaut-toml'
         title: 'Micronaut TOML'
         description: 'Micronaut TOML is the TOML configuration adapter for the Micronaut Framework.'
+      validation:
+        slug: 'micronaut-validation'
```

Comment:
micronaut validation does not have documentation: 

https://github.com/micronaut-projects/micronaut-validation/issues/36

Could you extract this from this PR and create a separate draft PR and add it to 4.0.0-M1